pages/register.py

```diff
@@ -37,6 +38,11 @@ def wait_for_page_to_load(self):
     def error_message(self):
         return self.find_element(*self._error_locator).text
 
+    def set_username(self, value):
```

Comment:
I see this method has no usage. Did you intend to use it somewhere?

Reply:
I used it for registering the users, but then decided not to include the registration script in my pull request. I'll remove this.